src/content/basics/byoc/index.md

```diff
@@ -0,0 +1,74 @@
+
+## What BYOC is good for {#benefits}
+
+Both on AWS and Azure, tenant cluster resources usually exist in an account (or in Azure terms: a subscription) separate from the one hosting the control plane resources. We configure a default account to use for all tenant clusters in an installation. Both accounts, the one for the control plane and the default one for tenant clusters, are in the customer's jurisdiction.
+
+**Note:** Some customers use the same account/subscription for both control plane and tenant cluster. This choice does not affect the capabilities described here.
+
+With BYOC you can have more fine-grained control over the accounts used by tenant clusters. Each Giant Swarm organization in an installation can have an individual configuration of which cloud provider account to use.
+
+The following two schemas illustrate the difference:
+
+![same-account](same-account.png)
+
+![same-account](separate-accounts.png)
+
+
+This enables use cases such as
+
+- Several teams, business units, or profit centers sharing an installation, where many or all of them run tenant clusters in their own cloud provider account, separate from each other.
+
+- An ISV, being the Giant Swarm customer, creating and giving access to tenant clusters in the name of a third party, in the third party's cloud provider account. The third party in this scenario has no relationship with Giant Swarm and needs no access to the Giant Swarm API or control plane.
+
+In both cases, customers benefit from simpler usage and cost allocation, plus a higher level of security through isolation. It can also help to make use of credits available in certain accounts.
+
+## Provider-specific mechanisms {#provider-specific}
+
+Details of the implementation differ between AWS and Azure.
+
+- On AWS, Giant Swarms uses two separate IAM roles to assume in order to act in the tenant cluster account: one for use by automation, one for technical support staff. Details on the exact permissions required are found in our guide on [preparing an AWS account to run Giant Swarm tenant clusters](/guides/prepare-aws-account-for-tenant-clusters/).
+
+- On Azure, one service principal is configured for Giant Swarm, used by automation and technical support staff. Detais are found in our guide on [preparing an Azure subscription to run Giant Swarm tenant clusters](/guides/prepare-azure-subscription-for-tenant-clusters/).
+
+## Details you should know about {#details}
+
+- Cloud provider account/subscription credentials are specified on the (Giant Swarm) **organization level**.
+
+- Cloud provider credentials are **immutable**. Once specified on an organization, cloud provider credentials cannot be modified or deleted. In order to switch to new cloud provider credentials you'll have to create a new organization and migrate to new cluster owned by that organization.
```

Comment:
> you'll have to create a new organization and migrate to new cluster owned by that organization.

'migrate to new clusters owned by that organization'?

Reply:
What a nie case for a suggestion :)

```suggestion
- Cloud provider credentials are **immutable**. Once specified on an organization, cloud provider credentials cannot be modified or deleted. In order to switch to new cloud provider credentials you'll have to create a new organization and migrate to new clusters owned by that organization.
```